templates/default/profile/path.sh.erb

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+DIRECTORIES=(<%= @directories.join(' ') %>)
```

Comment:
I think we can use a meaningful name rather than "directories", e.g. something like `path_default_folders`

Reply:
i don't share the same concern about the variable name, but if we're gonna change it let's opt for something containing 'directories' rather than 'folders', since that's the term used everywhere else.

Reply:
I totally agree with you that we must avoid ambiguous naming. Imho, in this case, there is no such ambiguity, but if you saw it, I'd rather name the variable `PATH_REQUIRED_DIRECTORIES` because, at the end, PATH can contains them and others.

Reply:
I was suggesting this renaming because in the path.sh.erb we were using a variable called `directories`, coming from elsewhere and if you look at this file alone is it not clear what this `directories` variable contains. Thanks for the renaming